newIDE/app/src/EventsSheet/ParameterFields/GenericExpressionField/FormatExpressionCall.js

```diff
@@ -85,9 +85,11 @@ export const getVisibleParameterTypes = (
 
 export const formatExpressionCall = (
   expressionInfo: EnumeratedInstructionOrExpressionMetadata,
-  parameterValues: ParameterValues
+  parameterValues: ParameterValues,
+  shouldConvertToString: boolean
```

Comment:
never a big fan of those loose booleans.
Can we instead have an options object, that has a `shouldConvertTostring` key? It's easier to read when we call the function to see `{ shouldConvertToString: true }` rather than just `true`